drivers/overlay/overlay.go

```diff
@@ -2554,11 +2554,18 @@ func (d *Driver) supportsIDmappedMounts() bool {
 }
 
 // SupportsShifting tells whether the driver support shifting of the UIDs/GIDs in an userNS
-func (d *Driver) SupportsShifting() bool {
+func (d *Driver) SupportsShifting(uidmap, gidmap []idtools.IDMap) bool {
 	if os.Getenv("_CONTAINERS_OVERLAY_DISABLE_IDMAP") == "yes" {
 		return false
 	}
 	if d.options.mountProgram != "" {
+		// fuse-overlayfs needs contiguous mappings to avoid https://github.com/containers/podman/issues/10272
```

Comment:
I have absolutely no idea how that record relates to `fuse-overlays` or contiguous mappings.

The comment there that is closest to an explanation talks about whether the invoking users’ host UID does, or does not, map into the container’s namespace …